src/internal/packager/git/gitea.go

```diff
@@ -177,7 +192,8 @@ func (g *Git) CreatePackageRegistryToken() (CreateTokenResponse, error) {
 
 	createTokensEndpoint := fmt.Sprintf("http://%s/api/v1/users/%s/tokens", tunnelURL, g.Server.PushUsername)
 	createTokensBody := map[string]interface{}{
-		"name": config.ZarfArtifactTokenName,
+		"name":   config.ZarfArtifactTokenName,
+		"scopes": []string{"read:user"},
```

Comment:
Are there any more desired scopes for the registry token?

Reply:
Since this is the push token it would need to have package write and read capabilities